db/seeds.rb

```diff
@@ -1,7 +1,10 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+require 'nokogiri'
+require 'open-uri'
+
+page = Nokogiri::HTML(open("https://quizlet.com/50590/1000-top-used-spanish-words-flash-cards/"))
+
+page.css('.term').each do |words|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.